src/Pipeline/Relay.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) 2009-2020 Daniele Alessandri
+ * (c) 2021-2023 Till Krüss
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Predis\Pipeline;
+
+use Predis\Connection\ConnectionInterface;
+use Predis\Response\ServerException;
+use Relay\Exception as RelayException;
+use SplQueue;
+
+/**
+ * Command pipeline that writes commands to the servers but discards responses.
+ */
+class Relay extends Pipeline
+{
+    /**
+     * These commands must be called on the
+     * client, not using `rawCommand()`.
+     *
+     * @var string[]
+     */
+    private $atypicalCommands = [
```

Comment:
Since we already have the connection, let's not define these twice. We could turn it into a public constant on the connection.